src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * added the list of constraint violations' parameters in `ConstraintViolationListNormalizer`
  * added support for serializing `DateTimeZone` objects
  * added a `deep_object_to_populate` context option to recursive denormalize on `object_to_populate` object.
+ * added support for denormalizing scalar values
```

Comment:
if this is a bug fix, this line should be removed
shouldn't the PR target 3.4 btw, for the same reason?

Reply:
When I was begin work on this PR I was guided on version from original issue. I don't mind rebase to 3.4, but don't know how to do that correctly. 3.4 requires support of PHP 5.5 that doesn't support array constants and constant visibility, so I will need to change this logic. And this changed logic will not be consistent with 4.3.
Сould you describe the correct procedure for me (so that I wouldn't make a mistake)?

Reply:
yes, it's quite common for an issue to be reported for an upper branch, it's our maintainer's job to figure out the lowest applicable branch...
we should use a `private static $scalarType` on 3.4, and I think that's all, isn't it?